### ISSUE TITLE
Order Creation: Accessibility → Create button should be disabled instead of hidden

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-ios#6067]
We don't show the Create button if nothing is changed. But this seems terrible for VoiceOver users. For someone who is just discovering the app, they may not know that there is a Create button in there that only appears after some conditions. I think we should show it but have it initially disabled. This will make it more discoverable and hopefully, the UI will be easier to grok. 

<img src="https://user-images.githubusercontent.com/198826/152426846-53e1592b-511e-4ab5-a606-ddd5f3e93f64.png" width="300"><img src="https://user-images.githubusercontent.com/198826/152426862-86ce6b4c-552e-49a4-ad9f-f2ac99540567.PNG" width="300">

**Comment by rachelmcr** [comment]
Closing this issue since we now always show the Create button (related: https://github.com/woocommerce/woocommerce-ios/pull/6308).
